staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -85,7 +85,7 @@ var (
 	//
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	groupVersions        = []schema.GroupVersion{auditv1alpha1.SchemeGroupVersion}
+	groupVersions        = []schema.GroupVersion{auditv1beta1.SchemeGroupVersion}
```

Comment:
@sttts used by the webhook initializer later on. As discussed in https://github.com/kubernetes/kubernetes/issues/48561 I think we should just do a hard switch for alpha to beta and use the beta group exclusively. We need to figure out a general way to let the client choose their preferred version later on.